src/Security/AgentChecker.php

```diff
@@ -12,9 +12,9 @@ class AgentChecker implements UserCheckerInterface
     public function checkPreAuth(UserInterface $user): void
     {
         if (!$user instanceof Agent) {
-            throw new CustomUserMessageAuthenticationException("Ce compte ne semble pas lié à un agent");
-        } else if (!$user->isActive()) {
-            throw new CustomUserMessageAuthenticationException("Ce compte agent a été désactivé");
+            throw new CustomUserMessageAuthenticationException('Ce compte ne semble pas lié à un agent');
+        } elseif (!$user->estValide()) {
+            throw new CustomUserMessageAuthenticationException('Ce compte agent a été désactivé');
```

Comment:
Le "elseif" n'est pas très utile, un "if" suffit.
Foncièrement ça ne change pas grand chose mais je trouve ça plus logique et lisible :
```suggestion
        }
        if (!$user->estValide()) {
            throw new CustomUserMessageAuthenticationException('Ce compte agent a été désactivé');
```

Reply:
J'abonde!